tutorial1/step6a.md

```diff
@@ -52,3 +52,10 @@ Extra options, that are structured like `-l` (or any other
 hyphen-letter combo) that turn something on/off e.g. a change in format, are 
 known as **flags**.  This isn't that important now, but it's a good thing to 
 know.
+
+________
+
+\* Note that the filesize listed in `ls` output for a directory path is _not_
+the total filesize of all the files in that directory. Instead, `ls` shows
+the filesize of the _reference_ to a _list_ of the files that that directory
+contains.
```

Comment:
I think this is a great improvement as it coves an easy point of confusion, but I think the language is still a little technical. I know it's not part of the main text, but talking about "filesize of the references to a list of files" doesn't add much clarity. However, I'm not sure what language would be better 🤔